src/StackDeploy/InstanceSpec.hs

```diff
@@ -77,3 +79,6 @@ mk name template = InstanceSpec
   , roleARN       = empty
   , ..
   }
+
+templateProvider :: Provider -> Template.Provider
+templateProvider provider = template <$> provider
```

Comment:
This could be `templateProvider = (template <$>)`

Reply:
yeah, fixing in followup. should make an hlint rule.